mapbox/services/analytics.py

```diff
@@ -0,0 +1,54 @@
+from mapbox.services.base import Service
+import dateutil.parser
+from dateutil.relativedelta import *
```

Comment:
Please avoid importing `*` from a module. Instead, do `dateutil.relativedelta import relativedelta`.

Also, generally, we organize imports into 3 sections: standard library modules, extra modules, and local package modules. Sorted by module name in each section, like

```
# (No standard library modules imported in analytics.py)

import dateutil.parser
import dateutil.relativedelta

from mapbox.services.base import Service
```

Finally, dateutil needs to be added to the `install_requires` list in setup.py. It is a dependency of boto or requests, but if we ever replaced those we'd no longer have a specification of our dependency on dateutil.